dap/GlobalMetadataStore.cc

```diff
@@ -748,35 +750,135 @@ GlobalMetadataStore::get_read_lock_helper(const string &name, const string &suff
  * true. When the MDSReadLock object goes out of scope, the read lock is
  * released.
  */
+// @Deprecated // 6.25.19 SBL
```

Comment:
Move the @deprecated into the comment for the method that starts with /** so that 'deprecated' will show up in the generated docs.